lib/Transform/ATenVisualGraph.cpp

```diff
@@ -704,6 +704,16 @@ struct ATenVisualGraphPass : public ATenVisualGraphBase<ATenVisualGraphPass> {
       fillPropertiesSliceOp(op2, std::move(props));
     } else if (auto op2 = dyn_cast<Torch::AtenConstantPadNdOp>(op)) {
       props.appendIntList("Attributes.padding", op2.pad());
+    } else if (auto op2 = dyn_cast<Torch::AtenMeanDimOp>(op)) {
+      // props.appendIntList("Attributes.padding", op2.pad());
```

Comment:
Why is most of the new code commented out? I usually don't like introducing commented out code, as it tends to get forgotten and bit-rot, but if there's a valid reason & precedence for it, then I'm fine with it.

Reply:
You're right - I had adopted the behavior of the code I copied whereas a `TODO` is clearer in intent. I'll adjust.